### PR TITLE
Add scaling='normalised' option to load_ard

### PR DESCRIPTION
**Pull request by cbur24**
### Proposed changes
Added a `scaling = 'normalised | raw'` option to `load_ard` to address the issue with native scaling on Landsat Collection 2 (a warning is also returned if scaling is set to 'raw').  I've also added a note in the Landsat Collections notebook about the proper way to scale C2.

### CODE REVIEW

**Comment by cbur24** [comment]
Switched default to `scaling='raw'` to avoid any issues with notebooks.  C2 re-scaling is now hard-coded inside `load_ard`.  We can revisit this issue when the definitive C2 is released.  Also updated the `collect_training_data` function in `classificationtools` to rescale when loading geomedians

File: Scripts/deafrica_classificationtools.py [hunk 1]
```diff
@@ -399,6 +399,7 @@ def get_training_data_for_shp(gdf,
     # TODO: Add support for other sensors
     if 'ga_ls8c_gm_2_annual' in products:
         ds = dc.load(product='ga_ls8c_gm_2_annual', **dc_query)
+        ds = ds * 2.75e-5 - 0.2
```
Comment (Kirill888) [inline Scripts/deafrica_classificationtools.py:402]: This needs to skip masking band.

File: Scripts/deafrica_datahandling.py [hunk 2]
```diff
@@ -478,6 +486,17 @@ def load_ard(dc,
...
+    if scaling=='normalised':
+        if product_type == 'c1':
+            ds = ds / 10000
+        if product_type == 's2':
+            ds = ds / 10000   
+    
+    if product_type == 'c2':
+        ds = ds * 2.75e-5 - 0.2
+        print("Scaling Landsat C2 using ds = ds * 2.75e-5 - 0.2")
+
```
Comment (Kirill888) [inline Scripts/deafrica_datahandling.py:499]: Same story as above, need to detect and not scale masking bands.

File: Scripts/deafrica_classificationtools.py (hunk 1 above)
Comment (cbur24) [reply Scripts/deafrica_classificationtools.py:402]: I didn't think the geomedian had a masking band?

File: Scripts/deafrica_datahandling.py (hunk 2 above)
Comment (cbur24) [reply Scripts/deafrica_datahandling.py:499]: Something like this?
```
if product_type == 'c2':
        print("Scaling Landsat C2 using ds = ds * 2.75e-5 - 0.2")
        
        if 'quality_l2_aerosol' in ds.data_vars:
            ds_data = ds[data_bands]
            ds_masks = ds[mask_bands]
            ds_data = ds_data * 2.75e-5 - 0.2
            ds = xr.merge([ds_data, ds_masks])
            ds.attrs.update(attrs)
        
        else:
            ds = ds * 2.75e-5 - 0.2
```

File: Scripts/deafrica_classificationtools.py (hunk 1 above)
Comment (Kirill888) [reply Scripts/deafrica_classificationtools.py:402]: sorry, I assumed this was generic load function

**Comment by cbur24** [comment]
Thanks for the reviews @fangfy @Kirill888

**Pull request merged**